internal/pkg/cli/env_init.go

```diff
@@ -228,25 +233,30 @@ func (o *initEnvOpts) Execute() error {
 			return fmt.Errorf("granting DNS permissions: %w", err)
 		}
 	}
-	// 1. Start creating the CloudFormation stack for the environment.
+	// 1. Attempt to create the service linked role if it doesn't exist.
+	// If the call fails because the role already exists, nothing to do.
+	// If the call fails because the user doesn't have permissions, then the role must be created outside of Copilot.
+	_ = o.iam.CreateECSServiceLinkedRole()
```

Comment:
Are those the only two failure cases? Is it okay not to fail here due to permissions issues without surfacing any error?

Reply:
I feel like it's fine because I don't' know what more we can do here for the user:
If the user doesn't have permission and the role was already created -> we don't want to fail.
If there was an internal error -> we don't want to fail.
For all other situations -> same behavior as today, it will print that the role is missing.